src/instructlab/lab.py

```diff
@@ -1059,9 +1059,8 @@ def get_files(pattern):
             )
             raise click.exceptions.Exit(1)
         if len(train_files) > 1 or len(test_files) > 1:
-            # pylint: disable=f-string-without-interpolation
             click.secho(
-                f"Found multiple files from `ilab generate`. Using the most recent generation.",
+                "Found multiple files from `ilab generate`. Using the most recent generation.",
```

Comment:
I don't mean to be picky but since you've put some effort into separating commits properly, I'd suggest you move the format-string in its commit.

Reply:
Sorry, perhaps I misunderstand but... this is part of removing the pylint disable: https://github.com/instructlab/instructlab/pull/1233/commits/2e65a8c9481050efb02f18496147ae2d8acdb0f4

Reply:
My bad! I overlooked that the formatting and pylint disable were related. Never mind :)